src/main/webapp/coffeeshop.html

```diff
@@ -12,10 +12,12 @@
   <body onload="loadStoreInfo(); determineLogin(); loadRatings();" id= "background">
     
 <nav id="navbar">
-   <a href="index.html" class="button">Home</a>
+   <a href="index.html" id="active" class="button">Home</a>
```

Comment:
Why is the home button the active tab for coffeeshop?

Reply:
My bad I didn't notice the mistake.